pkg/server/container_create_windows.go

```diff
@@ -299,7 +300,8 @@ func removeMount(s *runtimespec.Spec, dest string) {
 }
 
 func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxPid uint32, netnsPath string, config *runtime.ContainerConfig,
-	sandboxConfig *runtime.PodSandboxConfig, sandboxPlatform string, image *imagespec.Image) (*runtimespec.Spec, error) {
+	sandboxConfig *runtime.PodSandboxConfig, sandboxPlatform string, Image *imagestore.Image) (*runtimespec.Spec, error) {
```

Comment:
We shouldn't use upper-case variable name in Go for function arguments. Can we rename to something else?